views/Partials/Layout/_head.twig

```diff
@@ -16,3 +16,4 @@
     <link rel="stylesheet" href="{{site.theme.link}}/web/css/main.css" type="text/css" media="screen">
     <!-- Todo : add favicons -->
```

Comment:
Move todo to an external file instead of template to avoid missing delete during migration